docs/_templates/indexcontent.html

```diff
@@ -15,19 +15,19 @@
           <p class="links"><a href="{{ pathto("overview") }}#use-cases">Use cases</a> | <a href="{{ pathto("overview") }}#comparison">Comparison table</a> | <a href="{{ pathto("overview") }}#faq">FAQ</a></p>
     </div>
     <div class="item">
-        <h3><a href="{{ pathto("tutorials/first-steps") }}">Tutorials</a></h3>
-        <p class="description">The easiest way to get started.</p>
-        <p class="links"><a href="{{ pathto("tutorials/first-steps") }}">Store your data</a> | <a href="{{ pathto("tutorials/permissions") }}">Share between users</a> |  <a href="{{ pathto("tutorials/app-examples") }}">Application examples</a></p>
-    </div>
-    <div class="item">
         <h3><a href="{{ pathto("concepts") }}">Concepts</a></h3>
         <p class="description">Dive into the details: architecture and terminology.</p>
         <p class="links"><a href="{{ pathto("concepts") }}#buckets-collections-and-records">Buckets, Collections and Records</a> | <a href="{{ pathto("concepts") }}#inherited">Permissions inheritance</a> | <a href="{{ pathto("concepts") }}#groups">Groups</a></p>
     </div>
     <div class="item">
-        <h3><a href="{{ pathto("api/index") }}">APIs</a></h3>
-        <p class="description">An exhaustive look at the exposed HTTP endpoints.</p>
-        <p class="links"><a href="{{ pathto("api/index") }}#cheatsheet">Cheatsheet</a> | <a href="{{ pathto("api/index") }}#full-reference">Full reference</a> | <a href="{{ pathto("api/permissions") }}#permissions">Permissions</a> | <a href="{{ pathto("api/synchronisation") }}">Synchronisation</a></p>
+        <h3><a href="{{ pathto("contributing") }}">Community</a></h3>
+        <p class="description">Get help, contribute, and troubleshoot.</p>
+        <p class="links"><a href="{{ pathto("contributing") }}#how-to-contribute">How to contribute</a> | <a href="{{ pathto("contributing") }}#communication-channels">Communication channels</a> | <a href="{{ pathto("contributing") }}#hack">Hack</a> | <a href="{{ pathto("contributing") }}#troubleshooting">Troubleshooting</a></p>
+    </div>
+    <div class="item">
+        <h3><a href="{{ pathto("tutorials/first-steps") }}">Tutorials</a></h3>
+        <p class="description">The easiest way to get started.</p>
```

Comment:
Yeah, so I'd move the *Tutorials* section before the *Community* one. Actually to me, a natural section ordering would be:

1. Overview
2. Concepts
3. Tutorials
4. Configuration
5. Apis
6. Community (which could be renamed *Community and Support*)

Thoughts?